abis_mapping/utils/strings.py

```diff
@@ -0,0 +1,25 @@
+"""Provides string utilities for the package"""
+
+
+# Standard
+import re
+
+
+# Constants
+REGEX_STRIP = re.compile(
+    pattern=r"[\W_]+",
+    flags=re.IGNORECASE | re.UNICODE,
+)
+
+
+def sanitise(value: str) -> str:
+    """Capitalises and strips all non-alphanumeric characters from a string.
+
+    Args:
+        value (str): String to be stripped and capitalised
+
+    Returns:
+        str: The stripped and capitalised string
+    """
+    # Strip, Capitalise and Return
+    return REGEX_STRIP.sub("", value).upper()
```

Comment:
Haha, that got me into thinking about collision of meaning: English words that once concatenated mean something different. Example: together -> to get her :smile: 
(from https://www.reddit.com/r/wordplay/comments/ghqpeh/words_that_can_be_split_into_two_words/)

I don't think it is a problem here :smile: